recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set name = "PDFstream" %}
-{% set version = "0.4.3" %}
-{% set sha256 = "5ca61a43ecc176ac1dda5bcc82494b86b8398b7ca3428ceff6276d3ba13fc457" %}
+{% set name = "pdfstream" %}
+{% set version = "0.4.4" %}
+{% set sha256 = "d8a934446020e064171a3142a7e724b75c61ef6044356e96371a7ae1927bddb9" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://github.com/xpdAcq/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Looks right, same as in https://github.com/nsls-ii-forge/bluesky-feedstock/blob/14759c5e5837ed50db9625f26b229ed3065ecbcd/recipe/meta.yaml#L10.